sos/report/plugins/aap_gateway.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2024 Lucas Benedito <lbenedit@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class AAPGatewayPlugin(Plugin, RedHatPlugin):
+    short_desc = 'AAP Gateway plugin'
+    plugin_name = 'aap_gateway'
+    profiles = ('sysmgmt', 'ansible',)
+
+    packages = ('automation-gateway',
+                'automation-gateway-config')
+    commands = ('aap-gateway-manage',)
+    services = ('automation-gateway',)
+
+    def setup(self):
+        self.add_copy_spec([
+            "/var/log/nginx/access.log*",
+            "/var/log/nginx/error.log*",
+            "/var/log/supervisor",
+            "/etc/ansible-automation-platform",
```

Comment:
We're collecting this folder, but we already  the `AAPEDAControllerPlugin` collecting some of this as well, would there not be conflict here?

maybe, in the same way have `/etc/ansible-automation-platform/gateway`? if that is the main thing that is relevant for this plugin

Reply:
Based on the packages, commands, and services, the aap_gateway and aap_controller would not be executed in the same host. Those components only run on different nodes. With that, there wouldn't be conflicts
For the Controller plugin this path rarely contains data, while for the Gateway plugin, this is the primary place where things are logged.

Reply:
Sounds good to me, then.

Reply:
ack, sounds good to me